### PR TITLE
moved Button, Divider, Label, StylableBadge to a new deprecated folder

### PR DESCRIPTION
**Pull request by RashelleKosib**

### CODE REVIEW

File: docs/contribution-guide.md [hunk 1]
```diff
@@ -260,7 +260,7 @@ i.e:
 **themes/adi/button/button.st.css**
 ```css
 :import{
-    -st-from:'../../../components/button/buttons.st.css';
+    -st-from:'../../../components/deprecated/button/buttons.st.css';
```
Comment (shlomitc) [inline docs/contribution-guide.md]: this shouldn't be changed

File: docs/contribution-guide.md [hunk 2]
```diff
@@ -299,7 +299,7 @@ i.e:
 **themes/studio/button/button-3d.st.css**
 ```css
 :import{
-    -st-from:'../../../components/button/buttons.st.css';
+    -st-from:'../../../components/deprecated/button/buttons.st.css';
```
Comment (shlomitc) [inline docs/contribution-guide.md]: this shouldn't be changed

File: docs/contribution-guide.md (hunk 1 above)
Comment (RashelleKosib) [reply docs/contribution-guide.md]: @shlomitc  ill change it to a different component, like this:

-st-from:'../../../components/button/checkbox.st.css';

File: docs/contribution-guide.md [hunk 3]
```diff
@@ -260,7 +260,7 @@ i.e:
 **themes/adi/button/button.st.css**
 ```css
 :import{
-    -st-from:'../../../components/button/buttons.st.css';
+    -st-from:'../../../components/checkbox/checkbox.st.css';
     -st-defualt:Button
```
Comment (JoMarton) [inline docs/contribution-guide.md]: You need to completely change the example

**Pull request merged**